app/models/batch_context.rb

```diff
@@ -147,15 +148,12 @@ def output_dir_no_exists!
     FileUtils.mkdir_p(output_dir)
   end
 
-  # rubocop:disable Metrics/AbcSize
   def verify_staging_location
     return if errors.key?(:staging_location)
     return errors.add(:staging_location, "'#{staging_location}' not found.") unless File.directory?(staging_location)
 
     errors.add(:staging_location, "missing manifest: #{object_manifest_path}") unless File.exist?(object_manifest_path)
-    errors.add(:staging_location, "missing file manifest: #{file_manifest_path}") if using_file_manifest && !File.exist?(file_manifest_path)
   end
```

Comment:
**refactor:** the check for existence of a file manifest was moved to the new `verify_file_manifest` method added below